independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/handlers/ClientSetResponseEntityRestHandler.java

```diff
@@ -48,27 +54,32 @@ private boolean isAbortedWith(ClientRequestContextImpl requestContext) {
         return requestContext != null && requestContext.getAbortedWith() != null;
     }
 
-    private void setExistingEntity(Response abortedWith, ClientResponseContextImpl responseContext,
-            RestClientRequestContext restClientRequestContext) throws IOException {
-        Object value = abortedWith.getEntity();
-        if (value == null) {
-            responseContext.setEntityStream(null);
-            return;
+    private void propagateAbortedWithEntityToResponse(RestClientRequestContext restClientRequestContext) throws IOException {
```

Comment:
do I read it correctly that from this line there's no real change?

Reply:
That's right, it's just a minor refactoring